src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/env/EnvCleanupExtension.groovy

```diff
@@ -110,7 +110,7 @@ abstract class EnvCleanupExtension extends AbstractGlobalExtension implements Sp
                             it.dstSwitch.dpId == link.destination.switchId && it.dstPort == link.destination.portNo
                 }
                 if (!isl) {
-                    throw new IslNotFoundException("Wasn't able to find isl: $link")
+                    throw new IslNotFoundExpectedError("Wasn't able to find isl: $link")
```

Comment:
I don't think these 'expectedErrors' should be thrown. I also don't think IslNotFoundException should be renamed to IslNotFoundExpectedError. It's a pretty specific Exception that is meant to indicate a missing Isl, not an 'expected error'

Reply:
Seems like automatic refactoring mistake. Will fix.